cmd/configure.go

```diff
@@ -164,7 +164,7 @@ func runConfigure(configuration config.Config, flags *pflag.FlagSet) error {
 		workspace = config.DefaultWorkspaceDir(configuration)
 
 		// If it already exists don't clobber it with the default.
-		if _, err := os.Lstat(workspace); !os.IsNotExist(err) {
+		if _, err := os.Lstat(workspace); os.IsExist(err) {
```

Comment:
`os.Lstat` will return nil  if the path exist. `os.IsExist` requires error to be ErrExist which is why file existence is being checked with `!os.IsNotExist`; Lstat will always return ErrNotExist  if the path is not found otherwise nil (if there are no other errors).

Reply:
Yeah, I found that out lol. Thanks!